src/Core/Components/Badge/FluentBadge.razor.cs

```diff
@@ -12,15 +14,31 @@ namespace Microsoft.FluentUI.AspNetCore.Components;
 /// </summary>
 public partial class FluentBadge : FluentComponentBase
 {
+    private bool _isAttached => ChildContent is not null;
+
+    private string? _internalStyle => new StyleBuilder()
+        .AddStyle(" position: absolute; top: 0; left: 0; width: 100%; height: 100%; display: flex;")
```

Comment:
To increase performance, by replacing this long string with a CSS selector for all FluentBadges, you could add a CSS file containing something like this:
```css
span:has(fluent-badge[fuib]) {
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    display: flex;
}
```

You also need to add a `fuib` attribute to `fluent-badge` to avoid possible conflicts with other developments using this Badge element.

Reply:
You could do the same for the first `span` tag.